packages/api-rest/src/RestAPI.ts

```diff
@@ -204,16 +237,22 @@ export class RestAPIClass {
 	 * @return {Promise} - A promise that resolves to an object with response status and JSON data, if successful.
 	 */
 	head(apiName, path, init): Promise<any> {
-		const endpoint = this._api.endpoint(apiName);
-		if (endpoint.length === 0) {
-			return Promise.reject('API ' + apiName + ' does not exist');
+		try {
+			const apiConfig = this.getEndpointInfo(apiName, path);
+
+			const cancellableToken = this._api.getCancellableToken();
+
+			const initParams = Object.assign({}, init);
+			initParams.cancellableToken = cancellableToken;
+
+			const responsePromise = this._api.head(apiConfig, initParams);
```

Comment:
The other methods call this `apiInfo`.  Not a bug, but may be inconsistent?

```suggestion
			const apiInfo = this.getEndpointInfo(apiName, path);

			const cancellableToken = this._api.getCancellableToken();

			const initParams = Object.assign({}, init);
			initParams.cancellableToken = cancellableToken;

			const responsePromise = this._api.head(apiInfo, initParams);
```

Reply:
Thanks, I will rename that